splice/models.py

```diff
@@ -85,6 +85,8 @@ class Tile(db.Model):
     status = db.Column(db.String(16), nullable=False, server_default=u'unapproved')
 
     created_at = db.Column(db.DateTime(), nullable=False, server_default=db.func.now())
+    __table_args__ = (db.UniqueConstraint('adgroup_id', 'title', 'target_url', 'image_uri',
+                                          'enhanced_image_uri', 'type'),)
```

Comment:
This unique constraint also will cause migrations to fail on our production data